cmake/modules/CPM.cmake

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: MIT
+#
+# SPDX-FileCopyrightText: Copyright (c) 2019-2023 Lars Melchior and contributors
+
+set(CPM_DOWNLOAD_VERSION 0.40.8)
+set(CPM_HASH_SUM "78ba32abdf798bc616bab7c73aac32a17bbd7b06ad9e26a6add69de8f3ae4791")
```

Comment:
if this cmake/modules/CPM.cmake only exists to download a specific version of CPM.cmake from github, why not just check in that version of CPM.cmake here? seems unnecessary to download it over and over again for every ci build

Reply:
+1